src/store/assets/tokens.spec.ts

```diff
@@ -105,9 +105,9 @@ describe(mergeAssetBalances.name, () => {
         expect(tokenMerged.subBalance).toEqual(tokenUnanchored.balance);
       } else if (token && !tokenUnanchored) {
         expect(tokenMerged.balance).toEqual(token.balance);
-        expect(tokenMerged.subBalance).toEqual('0');
+        expect(tokenMerged.subBalance).toEqual(new BigNumber(0));
```

Comment:
Does this work? 🤔  What's being compared?

As BigNumber is a class `new BigNumber(0) !== new BigNumber(0)`

Reply:
It does pass, is there a better way to do this?

Reply:
I think it might be comparing the string `"[Object object]" === "[Object object]"`, but not sure. 

Perhaps, 

```
expect(tokenMerged.subBalance.toEqual(0)).toBeTruthy());
expect(BigNumber.isBigNumber(tokenMerged.subBalance).toBeTruthy());
```

Reply:
Found suggested to do: ```expect(tokenMerged.subBalance.toFixed()).toEqual(new BigNumber(0).toFixed());
        expect(BigNumber.isBigNumber(tokenMerged.subBalance)).toBeTruthy();```